lib/wprof/conf_wprof.rb

```diff
@@ -1,5 +1,16 @@
 module WProf
-  class Config
+  module Configuration
```

Comment:
Not a motive to reject, but we tend to write when multiple static methods exists as a:

```rb
class << self
end
```

block instead.

If you do so, you can use things like private